### PR TITLE
better smooth reading for docs of ef core querying

### PR DESCRIPTION
**Pull request by personball**
**Re-Translation** for docs of [ef core querying](https://docs.microsoft.com/zh-cn/ef/core/querying/).
Make it better smooth for reading in **Simplified Chinese (zh-CN)** .

### CODE REVIEW

**Comment by LisandroSu** [comment]
Hello, @personball 

Thank for your contribution. We are currently reviewing it. We will communicate the decision taken  accordingly. 
 
Thanks for your time and support on this.

Kind regards,

Microsoft C&E Localization Team

File: entity-framework/core/querying/async.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ ms.locfileid: "42993560"
...
 # <a name="asynchronous-queries"></a>异步查询
 
-当在数据库中执行查询时，异步查询可避免阻塞线程。 这有助于避免冻结胖客户端应用程序的 UI。 异步操作还可以增加 Web 应用程序的吞吐量，可以释放线程以在数据库操作完成前去处理其他请求。 有关详细信息，请参阅[使用 C# 异步编程](https://docs.microsoft.com/dotnet/csharp/async)。
+当在数据库中执行查询时，异步查询可避免阻止线程。这有助于避免冻结胖客户端应用程序的 UI。异步操作还可以增加 Web 应用程序的吞吐量，可以释放线程以在数据库操作完成前去处理其他请求。有关详细信息，请参阅[使用 C# 异步编程](https://docs.microsoft.com/dotnet/csharp/async)。
 
 > [!WARNING]  
 > EF Core 不支持在同一上下文实例上运行多个并行操作。 应始终等待操作完成，然后再开始下一个操作。 这通常是通过在每个异步操作上使用 `await` 关键字完成的。
 
-Entity Framework Core 提供了一组异步扩展方法，可用作执行查询并返回结果的 LINQ 方法的可选替代方法。 示例包括 `ToListAsync()`、`ToArrayAsync()`、`SingleAsync()` 等。对于部分 LINQ 操作（如 `Where(...)`、`OrderBy(...)` 等），没有对应的异步版本，因为这些方法仅用于构建 LINQ 表达式树，而未将查询发送到数据库中执行。
+Entity Framework Core 提供了一组异步扩展方法，可用作执行查询并返回结果的 LINQ 方法的替代方法。示例包括 `ToListAsync()`、`ToArrayAsync()`、`SingleAsync()` 等。对于部分 LINQ 运算符（如 `Where(...)`、`OrderBy(...)` 等），没有对应的异步版本，因为这些方法仅用于构建 LINQ 表达式树，而未将查询发送到数据库中执行。
```
Comment (personball) [inline entity-framework/core/querying/async.md]: Hi @LisandroSu , here "运算符" in Simplified Chinese usually means something like "+ -", I suggest "操作" for "operators",  as Where(...)  OrderBy(...) are LINQ methods.

File: entity-framework/core/querying/async.md [hunk 2]
```diff
@@ -13,12 +13,12 @@ ms.locfileid: "42993560"
 ---
 # <a name="asynchronous-queries"></a>异步查询
 
-当在数据库中执行查询时，异步查询可避免阻塞线程。 这有助于避免冻结胖客户端应用程序的 UI。 异步操作还可以增加 Web 应用程序的吞吐量，可以释放线程以在数据库操作完成前去处理其他请求。 有关详细信息，请参阅[使用 C# 异步编程](https://docs.microsoft.com/dotnet/csharp/async)。
+当在数据库中执行查询时，异步查询可避免阻止线程。这有助于避免冻结胖客户端应用程序的 UI。异步操作还可以增加 Web 应用程序的吞吐量，可以释放线程以在数据库操作完成前去处理其他请求。有关详细信息，请参阅[使用 C# 异步编程](https://docs.microsoft.com/dotnet/csharp/async)。
```
[the remark this answers is not in the document]
Comment (LisandroSu) [reply entity-framework/core/querying/async.md]: Hello, @personball thank you for your feedback. I sent your comments to the Linguistic Team. We will let you know as soon as we have an answer.

Regards,
Microsoft C&E Localization Team

File: entity-framework/core/querying/async.md (hunk 1 above)
Comment (LisandroSu) [reply entity-framework/core/querying/async.md]: Hello, @personball thank you for your feedback. I sent your comments to the Linguistic Team. We will let you know as soon as we have an answer.

Regards,
Microsoft C&E Localization Team

File: entity-framework/core/querying/async.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (personball) [reply entity-framework/core/querying/async.md]: @LisandroSu  see https://www.microsoft.com/en-us/language/Search?&searchTerm=blocking&langID=Chinese%20Simp.&Source=true&productid=All%20Products

**Comment by personball** [comment]
resolve conflicts

**Comment by personball** [comment]
@LisandroSu Is this pr in progress or rejected?

**Pull request merged**